### PR TITLE
Refine configuration-cache report after the addition of configuration inputs

### PR DESCRIPTION
**Pull request by eskatos**
Fix the report layout moving inputs into their own tab, remove now irrelevant filter buttons, rework the title/summary of the report to account for inputs, display the requested tasks for the report so they are easier to identify, plus various enhancements.

image

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test ACC

**Comment by eskatos** [comment]
@bot-gradle test ACC

**Comment by bamboo** [comment]
The report looks great! 👍

File: subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt [hunk 1]
```diff
@@ -186,45 +165,74 @@ object ConfigurationCacheReportPage : Component<ConfigurationCacheReportPage.Mod
...
 
     private
-    fun Model.problemOrProblems() =
-        if (totalProblems == 1) "problem was" else "problems were"
+    fun found(count: Int, what: String) =
+        "${count.toStringOrNo()} ${what.sIfPlural(count)} ${wasOrWere(count)} found"
+
+    private
+    fun Int.toStringOrNo() =
+        if (this != 0) toString()
```
Comment (bamboo) [inline subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt:213]: minor 💄 - the positive test would read slightly better `if (this == 0) "No" else toString()`

**Review by bamboo — APPROVED** [review]
🎉

File: subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt (hunk 1 above)
Comment (eskatos) [reply subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt:213]: I wrote it with a positive test at first but changed it so it reads the same as the function name.

Comment (eskatos) [reply subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt:213]: What about?
```kotlin
fun Int.toStringOrNo() =
     if (this > 0) toString()
     else "No"
```

Comment (bamboo) [reply subprojects/configuration-cache-report/src/main/kotlin/ConfigurationCacheReportPage.kt:213]: Then I'd fear negative values could be missed if we ever introduce a related bug :D

Not big deal, it's good enough as it is 👍

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**